### DISCUSSION TITLE
Caching Variables

### DISCUSSION THREAD

**Discussion by qhtjd03** [ScratchAddons/ScratchAddons#4788] [Ideas]
https://scratch.mit.edu/discuss/topic/87548/?page=3#post-765518
Seven years ago, a user succeeded in caching a scratch variable. If so, is this still possible?

**Comment by BroJac5246** [comment]
Potentially, using a browser extension. It could probably be implemented using Scratch Addons, but it's important to remember that it would only affect users of that extension.
